setup.py

```diff
@@ -270,6 +270,9 @@ def write_version(filename: str = str(AIRFLOW_SOURCES_ROOT / "airflow" / "git_ve
 atlas = [
     "atlasclient>=0.1.2",
 ]
+backports = [
+    "backports.zoneinfo>=0.2.1;python_version<'3.9'",
+]
```

Comment:
Maybe we should just add it into the main dependencies? By the same way as we done for `importlib` backports?

https://github.com/apache/airflow/blob/55fbdfea4235569c91942e6b2227090366e0e75a/setup.cfg#L106-L107

Reply:
Yes, you are absolutely right.